litestar/_openapi/schema_generation/examples.py

```diff
@@ -14,9 +14,9 @@
 from polyfactory.utils.predicates import is_union
 from typing_extensions import get_args
 
+from litestar.contrib.pydantic.utils import is_pydantic_model_instance
```

Comment:
Would this cause the `MissingDependencyError` to be raised if pydantic is not installed?

Reply:
Good catch. Yeah, it would. Oh my, this whole stuff is a mess. I'll try to figure out a way that actually works correctly and is a bit nicer :)

Reply:
I fixed it. I think :grimacing: